src/MissionManager/MissionController.cc

```diff
@@ -848,12 +927,6 @@ void MissionController::_deinitVisualItem(VisualMissionItem* visualItem)
     disconnect(visualItem, 0, 0, 0);
 }
 
-void MissionController::_itemCoordinateChanged(const QGeoCoordinate& coordinate)
-{
-    Q_UNUSED(coordinate);
-    _recalcWaypointLines();
-}
-
```

Comment:
You're just changing the coordinate in the model now right when the coordinate changes as opposed to rebuilding? Is that correct? This was specifically what didn't work before. I needed to do a full rebuild and model change to get coordinates to update.